pages/common/ssh-agent.md

```diff
@@ -0,0 +1,18 @@
+# ssh-agent
+
+> Spawn an ssh-agent process.
+> The ssh-agent will hold your ssh key decrypted in memory until evicted.
+> Add and manage keys with `ssh-add`.
+
+- Start an SSH Agent for the current shell:
+
+`eval $(ssh-agent)`
+
+- Kill the current agent:
+
+`ssh-agent -k`
+
+- Manually connect to an existing agent (rare):
+
+`export SSH_AUTH_SOCK={{ssh-agent socket}}`
+`export SSH_AGENT_PID={{ssh-agent pid}}`
```

Comment:
Just a suggestion per my previous comment.
```suggestion
`export SSH_AUTH_SOCK={{ssh_agent_socket}} SSH_AGENT_PID={{ssh_agent_pid}}`
```

Reply:
We can do this if you want, I don't think exporting two variables on one line is clearer though.

Reply:
Sorry for aking so lonig to get back to you.

True, but what @zdroid was trying to say is that in tldr pages we only have 1 line of command for each example. In addition, I'm not really sure of the usefulness of this example as it is, since the `eval` one does this for you.

tldr pages aren't supposed to be _exhaustive_ - just include the most commonly used and most useful examples. So given the either existing example, I'm not sure we need this example unless you've got a good use-cas for not using the `eval` one.

Reply:
Ok I will remove it. We can always add it back if that makes sense or a user asks for it!